### PR TITLE
Updated test cases

### LINKED ISSUE #27: Unintented universal offset?
It needs to be verified that the 90 degree orientation offset is universal (as implemented) or specific to skydb. If it's the later, the code should be altered to accept a desired offset.

https://github.com/soravux/skylibs/blob/adb148d849fe8b5c99c89958fb663c7114a60fda/hdrtools/sunutils.py#L100

### PR DESCRIPTION
**Pull request by IanMaquignaz**
resolves #27 

- Fixes typos in documentation
- Adds two new projections
- Adds updated test cases for new projections and pySolar tests.

Notes:
- There is an inherent ambiguity to using angles. If care is not taken, it possible to loose track of the orientation of the azimuth, as the origin is transformed between theory and cartesian space.

### CODE REVIEW

**Comment by IanMaquignaz** [comment]
@soravux , thoughts?

**Comment by soravux** [comment]
Thanks for the typo fixes in the documentation.

The "new projection" is not actually doing a new projection but performing a conversion from `world2latlong`'s normalized polar coordinates on the sphere (the `u,v` representation used throughout skylibs) to actual angles in a different coordinate system than the rest of the library. This conversion can already be achieved by using the current methods (return values in radians and coherent with skylib's coordinate system):
```u, v = world2latlong(x, y, z)
azimuth = np.pi*(2*u - 1)
elevation = np.pi*v
```
and
```
u = (1 + azimuth/np.pi)/2
v = elevation/np.pi
x, y, z, _ =  latlong2world(u, v)
```
Let's try not to have duplicated code to maintain. We should add this conversion from normalized polar coordinates to radians described in the `README.md`, to clarify the conversion to and from our coordinates system.

Supporting multiple representations through parameters such as `degrees` and `orientation_angles2world` is a maintenance burden, we need to work with a single representation where we can (i.e., only in radians, and only with the coordinates system presented in the readme). It should be the caller's task (in this case, `sunutils.py`) to make sure the data fits the representation when calling the function.

`world2elevationAzimuth` crashes if `degrees == False`, and the returned `elevation` is actually a zenith angle, not the elevation angle.

**Comment by IanMaquignaz** [comment]
@soravux  all cleaned up!

File: hdrtools/sunutils.py [hunk 1]
```diff
@@ -50,58 +51,88 @@ def findBrightestSpot(image, minpct=99.99):
...
-    azimuth = solar.get_azimuth(latitude, longitude, time, elevation)
-    elevation = solar.get_altitude(latitude, longitude, time, elevation)
+    zenith, azimuth = sunPosition_pySolar_zenithAzimuth(
+        latitude, longitude, 
+        time, 
+        elevation
+    )
+  
+    # Fix orientation of azimuth
+    azimuth = -(azimuth - (np.pi/2))
```
Comment (IanMaquignaz) [inline hdrtools/sunutils.py:112]: Is this redundant?

File: hdrtools/sunutils.py [hunk 2]
```diff
@@ -50,58 +51,88 @@ def findBrightestSpot(image, minpct=99.99):
...
-    azim = solar.get_azimuth(latitude, longitude, time_, elevation)
-    alti = solar.get_altitude(latitude, longitude, time_, elevation)
+    # Find azimuth and elevation from pySolar library.
+    azimuth = solar.get_azimuth(latitude, longitude, time, elevation)
+    altitude = solar.get_altitude(latitude, longitude, time, elevation)
 
     # Convert to radians
-    azim = np.radians(-azim)
-    elev = np.radians(90-alti)
+    azimuth = (np.pi/2) + np.deg2rad(-azimuth)
```
Comment (IanMaquignaz) [inline hdrtools/sunutils.py:85]: Is this redundant?

**Review by IanMaquignaz — COMMENTED** [review]
Given it is immediately inverse, it the azimuth offset really necessary?

I assume it orients the angle correctly for representation in radians (assume, as I have no reference for validation). In terms of the code, it is immediately inverted, so it has little value.

**Pull request merged**